pkg/operator/ceph/controller/controller_utils.go

```diff
@@ -116,6 +118,19 @@ func LoopDevicesAllowed() bool {
 	return loopDevicesAllowed
 }
 
+func SetEnforceHostNetwork(data map[string]string) {
+	val := k8sutil.GetValue(data, EnforceHostNetworkSettingName, EnforceHostNetworkDefaultValue)
+	var err error
+	if cephv1.EnforceHostNetwork, err = strconv.ParseBool(val); err != nil {
+		cephv1.EnforceHostNetwork = false
+		logger.Warningf("failed to parse value %q for %q. assuming false value", val, EnforceHostNetworkSettingName)
+	}
+}
+
+func EnforceHostNetwork() bool {
+	return cephv1.EnforceHostNetwork
```

Comment:
This is such a simple wrapper, why not just have all the callers reference `cephv1.EnforceHostNetwork` directly?

Reply:
I did in network.go and network_test.go. for other cases, I found the use of the getter more appealing.

Reply:
I also directly accessed `cephv1.EnforceHostnetwork` directly in `cmdreporter.go` to avoid an import cycle.

Reply:
What about moving this getter to the cephv1 package? Then you can change it to a private var (lower case) `enforceHostNetwork` so access outside the package is only through the methods. Currently, anyone could choose to access the public var directly and skip the helpers.